templates/graph.html

```diff
@@ -0,0 +1,21 @@
+{% extends "page.html" %}
+{% block head %}
+  <title>Home page | F# Snippets</title>
+{% endblock %}
+
+{% block customPageScripts %}
+<script type="text/javascript" src="https://www.google.com/jsapi"></script>
```

Comment:
@TahaHachana is there an easy way to generate this script snippet block?
imag.Js is missing the `google.load("visualization", "1", { packages: ["corechart"] })` part and doesn't load https://www.google.com/jsapi